Adafruit_SSD1306.cpp

```diff
@@ -1090,3 +1090,27 @@ void Adafruit_SSD1306::dim(boolean dim) {
   ssd1306_command1(dim ? 0 : contrast);
   TRANSACTION_END
 }
+
+/*!
+    @brief  Set display brightness.
+    @param  contrast
+            255 for full brightness.
+    @return None (void).
+    @note   Max contrast value is 255
+*/
+void Adafruit_SSD1306::setBrightness(uint8_t contrast) {
+  // low brigthness can be set with the help of precharge
+  TRANSACTION_START
+  if(contrast>255) {
+    contrast = 255;
+  }
```

Comment:
Could just use the constrain function
```suggestion
contrast = constrain(contrast, 0, 255);
```